wg-naming/README.md

```diff
@@ -0,0 +1,70 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Naming Working Group
+
+
+## Stakeholder SIGs
+* SIG Architecture
+* SIG Contributor Experience
+* SIG Docs
+
+
+
+## Organizers
+
+* Celeste Horgan (**[@celestehorgan](https://github.com/celestehorgan)**), CNCF
+* Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**), VMware
+* Zach Corleissen (**[@zacharysarah](https://github.com/zacharysarah)**), Linux Foundation
+
+## Contact
+- Slack: [#wg-naming](https://kubernetes.slack.com/messages/wg-naming)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-naming)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Fnaming)
+<!-- BEGIN CUSTOM CONTENT -->
+
+**The following section will be reworked and formalized as a charter once the
+Working Group has been approved by the Steering Committee.**
+
+### Goals
+
+- Evaluate language and naming choices within the Kubernetes organization, with
+  a specific initial focus of:
+  - Removing language which cannot be fully separated from racist, sexist or
```

Comment:
This list isn't complete (missing ableist and transphobic at least). Rather than try to list every group, what about just saying "Removing language which alienates unpriviliged groups of people" or something like that.

Reply:
"Removing language which causes harm to protected classes and under-represented individuals" maybe?

Reply:
> Rather than try to list every group, what about just saying "Removing language which alienates unpriviliged groups of people" or something like that.

This seems like a good idea.

Reply:
> "Removing language which causes harm to protected classes and under-represented individuals" maybe?

One more thought, I think our standard should be being excellent to each other, not just lack of harm (which is why I used the word "alienates")... I guess running with that thought, we could try to phrase in the positive, like "Ensure all language is inclusive of protected classes and under-represented individuals, by removing any that is not." (Honestly I'm not sure if the classes of people I'd want to be excellent to are all "protected" and probably not in all jurisdictions, so I'd go for something even broader, but that's just me personally.)

Anyway I don't actually have a strong opinion other than an incomplete list is probably not good.

Reply:
I appreciate the drive for language that's more inclusive and oriented towards positive outcomes. I appreciate that "alphabet soup" lists often seem cumbersome and do shift over time as awareness and specificity grow. 

There's a reason why lists get longer rather than shorter. It matters to be specific. "Diversity and inclusion" are empty words. The goal of this WG is to stop harm. The reason why this WG exists is because the generic language we already have doesn't stop harm.

Generic language obscures harm rather than cultivating good action; note _action_, not intent. I assume everyone's intent is only for the best; and intent is irrelevant to the discussion. We can't control people's intent, or even assess it meaningfully. We can only assess actions, and so we have to be specific about what actions cause harm.

When people don't know that they specifically are protected and welcome, they tend to assume they're not--whether for race, gender, sexual orientation, or any other group.

I need to be clear that ongoing "debate" here does more harm than good. Endless reviews and decision by committee is a tactic for stalling meaningful change. We will inevitably fall short with whatever language we use, and we'll need to iterate to improve it. So it goes. 

As for actual suggestions, maybe:
```suggestion
  - Removing barriers to contribution and adoption by replacing harmful language with neutral terms whenever possible, including but not limited to language linked to racism, sexism, homophobia, transphobia, ableism, or discrimination against any protected or historically underrepresented group.
```

Reply:
@zacharysarah i'd prefer to think that everyone reviewing this proposal is doing it in good faith. I don't think we have reached the "Endless reviews" threshold. I don't saying things like "decision by committee is a tactic for stalling meaningful change" is fruitful to the effort we are involved in. So far it's taken the "normal" route/time. So let's focus on what we need to do together in good faith.

Reply:
To be clear, this has been a documented issue in our community for over a year (2 years?) so it's reasonable to skew towards action. I think the update suggested above is a good one, and balances specificity with some placeholders for evolving awareness.

Reply:
@jdumars is the "stalling" (words from Zach) for 2 years (your count) because of active opposition? or is it because of lack of hands to do the work?

Reply:
@dims Let's please refocus on the goal of approving a working group.

Reply:
I agree with @dims that everyone here is working in good faith towards the goal of approving a working group. Like any organization the size of Kubernetes, we have governance processes to go though, and review/approval of the goals and criteria of a WG would be the same no matter the topic the WG is attempting to tackle.

I am completely good and supportive of @zacharysarah's suggestion around revised language for this point.

Reply:
We will take @zacharysarah's suggestion then, and consider this discussion thread closed.